### DISCUSSION TITLE
flex-col and flex-wrap

### DISCUSSION THREAD

**Discussion by vinny74man** [tailwindlabs/tailwindcss#8887] [Help]
Hello everyone can you help me I'm trying this:

`<div class="flex flex-col md:flex-wrap">
    <div>
        ....
    </div>
    <div>
        ....
    </div>
</div`>
 but it doesn't work, even with display md the visualization is always in column.

Thank you

Sorry this is an online translation

**Comment by arisac** [comment]
I don't understand what you are trying to do. 
You don't just paste a code and saying it's not working without saying how you want it to work like.

Maybe you want to know what is `flex-row` and `flex-col` and change them in different screen breakpoints like this? https://play.tailwindcss.com/wt6q17d7sb

**Comment by vinny74man** [comment]
Yes I want that if the display is smaller than md the flex-col class works otherwise the flex-wrap class, if I do the same thing with flex-col and flex-row it works, but with flex-col and flex-wrap it doesn't.
At least I can't get it to work.

Thank you
Sorry this is an online translation

**Reply by arisac** [reply]
So it's like https://play.tailwindcss.com/wt6q17d7sb?

You need `flex flex-col md:flex-row md:flex-wrap`. 

- `md:flex-row` to change the flex direction from col to row 
- and `md:flex-wrap` to make make it wrap.

**Comment by vinny74man** [comment]
OK thank you very much

**Reply by arisac** [reply]
@vinny74man please close this discussion and mark as answered so other people don't need come in to see if you still help.
